src/test/CMakeLists.txt

```diff
@@ -424,3 +426,61 @@ install(
   TARGETS triton_repo_agent_test
   RUNTIME DESTINATION bin
 )
+
+
+if(${TRITON_ENABLE_METRICS})
+  #
+  # Unit test for Generic Metrics API
+  #
+  add_executable(
+    metrics_api_test
+    metrics_api_test.cc
+    ../metric_family.cc
```

Comment:
If the unit test only interact with Triton core via C APIs, I don't think you need to include these files